episodes/editing.md

```diff
@@ -230,9 +230,19 @@ outlines, aggregated figures, and slides would live in this folder.
 
 ## `learners/`
 
-All the extras the learner would need, mostly a setup guide and glossary would
+All the extras the learner would need, mostly a setup guide and glossary 
 live here.
 
+The glossary page is populated from the `reference.md` file in this folder.
+The format of the `reference.md` file is two lines for each term, the first
+includes the term to be defined and then the second line starts with a ":"
+and a space then the definition.
+i.e.
+```markdown
+term
+: definition
+```
+
```

Comment:
This would allow it to render so that people can see what definition lists look like

````suggestion
```

term
: definition

````